.github/workflows/ci-testing-deploy.yml

```diff
@@ -384,6 +384,8 @@ jobs:
         run: ./ci/helpers/show_system_versions.bash
       - name: install
         run: ./ci/github/unit-testing/storage.bash install
+      - name: typecheck
+        run: ./ci/github/unit-testing/storage.bash typecheck
       - name: test
         run: ./ci/github/unit-testing/storage.bash test
```

Comment:
do not forget to add that ``test`` runs even if ``typecheck`` fails. See the others ``typecheck``)

Reply:
hmm I copy pasted that. so I will then fix it everywhere